website_contact_lastname/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Website Contact Lastname",
+    "version": "15.0.1.0.0",
+    "category": "Website",
+    "website": "https://github.com/OCA/website",
+    "author": "Sygel Technology," "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "auth_signup",
+        "website_account_fiscal_position_partner_type",
```

Comment:
Isn't this too specific for the task?

Reply:
Sorry, but I don't think I get what you mean in this point.

Reply:
That this dependency is too specific and not everyone will be interested in pulling it to enjoy this new feature

Reply:
then we would have to carry out 3 separate modules. Since we have made the extension for the secondname. I don't see any harm in indicating on the website whether the client is a company or an individual for the correct treatment of the fields (the addition of the dependency).

Reply:
I agree. The module has been designed considering that a customer can be B2B or B2C. Considering this fiscal position type, the dependency is a must